test/common/markdown/markdown-link.tests.ps1

```diff
@@ -0,0 +1,111 @@
+﻿# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "Verify Markdown Links" {
+    BeforeAll {
+        # WARNING: Keep markdown-link-check pinned at 3.7.2 OR ELSE...
```

Comment:
What was I thinking when I wrote the `OR ELSE...` part haha
It will only break tests (not end the world)